integration/spark/app/build.gradle

```diff
@@ -202,9 +202,9 @@ task copyDependencies(type: Copy) {
 task integrationTest(type: Test) {
     // integrationTest depends on shadowJar which dependsOn test
     // exclude all the tests and add manually tagged with integration-test
-    test {
-        exclude "**"
-    }
+//    test {
```

Comment:
?

Reply:
it was done so the tests were not omitted in `app`, was supposed to be rolled back